tests/logictest/suites/base/02_function/02_0002_function_cast

```diff
@@ -215,6 +215,12 @@ SELECT to_date(to_timestamp(1640019661000000)) = to_date('2021-12-20');
 ----
 1
 
+statement query B
+SELECT to_date('2022-02-03T00:00:00+09:00') = to_date('2022-02-02');
```

Comment:
Time strings lose hours, minutes, and seconds when converting to date, which can lead to incorrect equations.
For example:
```sql
SELECT to_date('2022-02-03T02:00:00+09:00') = to_date('2022-02-02');
```